mantidimaging/core/utility/command_line_path.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 2021 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+class CommandLinePath:
+    _instance = None
+    images_path = ""
+
+    def __new__(cls, path: str = ""):
+        """
+        Creates a singleton for storing the path from the command line argument.
+        """
+        if cls._instance is None:
+            cls._instance = super(CommandLinePath, cls).__new__(cls)
+            cls.images_path = path
```

Comment:
Can you validate that the path exists here, I think it would be good to exit without starting the GUI if path does not exist.